docs/product/build-distribution/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Build Distribution
+sidebar_order: 137
+description: Distribute app builds to internal teams and beta testers.
+---
+
+<Include name="build-distribution/ea" />
+
+Build Distribution enables you to securely distribute app builds to your internal teams and beta testers. Upload builds from CI to streamline your distribution workflow, manage access control, and track installation analytics.
+
+### CI Integration
+
+Integrate Build Distribution into your CI pipeline to automatically distribute builds to your teams.
```

Comment:
Do we need to mention anything else?
There's no difference if a build is uploaded from a local build vs a CI build for the purposes of size analysis. There isn't any github integration either.